Leaflet.VectorTiles.js

```diff
@@ -655,7 +776,7 @@ L.VectorTiles = L.GridLayer.extend({
    * @returns {L.Path}
    * @private
    */
-  _geojsonToLayer(feature, id) {
+  _geojsonToLayer(feature) {
```

Comment:
Two issues here:
* The coordinate array of `MultiPolygon` is a level deeper than `Polygon`.
* It's very elegant to use `map()` in a nested loop,  but it may lead to frequent function creations and use more memory (see [comparison](http://stackoverflow.com/questions/42728819/array-foreach-and-closure)).